src/canopy_server/app.py

```diff
@@ -184,7 +186,7 @@ async def upsert(
 
 
 @app.post(
-    "/context/delete",
+    f"/api/{API_VERSION}/context/delete",
     response_model=SuccessDeleteResponse,
     responses={500: {"description": "Failed to delete documents"}},
 )
```

Comment:
@miararoy if you're adding versioning - than **all** API endpoints should have version, including `/health` etc (it has a structured return value, that may change between versions)

Reply:
done